src/js/actions/event/data.js

```diff
@@ -0,0 +1,47 @@
+export const GET_EVENT = 'GET_EVENT';
+export const GET_EVENT_REQUEST = 'GET_EVENT_REQUEST';
+export const GET_EVENT_SUCCESS = 'GET_EVENT_SUCCESS';
+export const GET_EVENT_FAILURE = 'GET_EVENT_FAILURE';
+
+const event1 = {
+  event_id: 1,
+  host_user_id: 1,
+  name: 'Lounge party',
+  description: 'Celebrating life',
+  is_poll: true,
+  is_edited: false,
+  _invitees: [2],
+  _what: ['dancing', 'skydiving'],
+  _where: ['Forest', 'Camping'],
+  _when: ['2017-01-03T00:00:00.000Z', '2017-02-14T00:00:00.000Z']
```

Comment:
@minaorangina maybe its worth refactoring _when:[{date: , time: }] to be only timestamp and when we show it on summary page we will convert from timestamp to readable format for time and date. It might be more work doing conversion from date to timestamp and from time to timestamp. I have noticed that when you pick the date from DATEPICKERIOS - the format is date that you picked and time. But also you pick a time as well which is only time. Let me double check that.